quill-core/src/main/scala/io/getquill/context/ContextMacro.scala

```diff
@@ -65,7 +65,7 @@ trait ContextMacro extends Quotation {
 
         ProbeStatement(idiom.prepareForProbing(string), c)
 
-        c.info(string)
```

Comment:
If you want to see SQL at compile time, you should edit your IDE configuration.

I think the SQL log should not be issued at runtime.